### ISSUE TITLE
can not use npm package

### ISSUE THREAD

**Issue by choehn-signogy** [toolisticon/ssl-hostinfo-prometheus-exporter#471]
The installed rpm package seems to be broken. It can not be started like described.
It seems that app.js is missing a '#!/usr/bin/env node' at the beginning.

(
maybe it should be in index.js, but 
/usr/bin/ssl-hostinfo-prom is pointing to /usr/lib/node_modules/@toolisticon/ssl-hostinfo-prometheus-exporter/app.js
)

Furthermore, the installed npm package is missing the whole ./lib folder which is present after git clone
That leads to another error when running node index.js / node app.js

## What did you do?

$ sudo npm install --location=global @toolisticon/ssl-hostinfo-prometheus-exporter
$ export URLS_TO_CHECK=app1.sample.com,app2.sample.com
$ export CRON='0 * * * * *'
$ export LOG_LEVEL=ERROR
$ export CONSOLE_LOG=false
$ ssl-hostinfo-prom

## What did you expect to happen?

A regular startup of the program.

## What happened instead?

A big error message:

$ ssl-hostinfo-prom
```
/usr/bin/ssl-hostinfo-prom: line 1: use strict: command not found
/usr/bin/ssl-hostinfo-prom: line 2: syntax error near unexpected token `('
/usr/bin/ssl-hostinfo-prom: line 2: `const CronJob = require('cron').CronJob;'
```

$ cd /usr/lib/node_modules/@toolisticon/ssl-hostinfo-prometheus-exporter/
$ node index.js 

```
node:internal/modules/cjs/loader:988
  throw err;
  ^

Error: Cannot find module './lib/config'
Require stack:
- /usr/lib/node_modules/@toolisticon/ssl-hostinfo-prometheus-exporter/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:985:15)
    at Function.Module._load (node:internal/modules/cjs/loader:833:27)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/usr/lib/node_modules/@toolisticon/ssl-hostinfo-prometheus-exporter/index.js:3:16)
    at Module._compile (node:internal/modules/cjs/loader:1155:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/lib/node_modules/@toolisticon/ssl-hostinfo-prometheus-exporter/index.js'
  ]
}
```

## Your Environment

Ubuntu 22.04.1 **LTS**
NodeJS v16.18.0
NPM 8.19.3

**Comment by hypery2k** [comment]
thx for reporting the issue. Can you give `@toolisticon/ssl-hostinfo-prometheus-exporter@next` a try?

**Comment by hypery2k** [comment]
should be fixed with version `2.1.7` 
If it's not working, feel free to reopen the issue
